extensions/wikia/Rail/styles/RailElement.scss

```diff
@@ -232,51 +228,51 @@ $color-rail-background: mix($color-page, $black, 90%);
 
                 .carousel-container {
                     left: 16px;
-                    width: 260px;
+                    width: 220px;
                 }
 
                 .since, .chatter {
-                    width: 32px;
+                    width: 30px;
                 }
+            }
+        }
+    }
+}
 
-                .UserStatsMenu {
-                    outline: rgba($color-divider, .3) solid 2px;
-                    width: 220px;
+.UserStatsMenu .new-chat {
```

Comment:
If that's really temporary, then I'm fine with that as long as you add a `TODO` which will be resolved at the right moment.